examples/uv_actor.lua

```diff
@@ -0,0 +1,48 @@
+-- Use LibUV event loop in actor thread
+--
+-- !NOTE! I can not figure out why this does not work
+--  with `inproc` transport so I use `tcp` to communicate
+--  with actor thread
```

Comment:
try making the child send a message to the parent instead of the other way around?

see "slow joiner" on http://zguide.zeromq.org/page:all